### PR TITLE
Add sharing your work

### PR DESCRIPTION
**Pull request by pbchase**
@tlstoffs, could you look at the 'Sharing your work' section of guide_for_admins_and_devs.md

@ChemiKyle, could you verify the authors display correctly on you machine? They look like this on mine:

authors

### CODE REVIEW

**Comment by pbchase** [comment]
I realize now this should have been 2 PRs. Sorry. I'll try not to do this next time.

**Comment by ChemiKyle** [comment]
Authors render properly on my machine, I'll skim the grammar once more just to be safe and then merge.

File: docs/guide_for_admins_and_devs.md [hunk 1]
```diff
@@ -166,24 +166,17 @@ Wherever they test, REDCap Admins must test the new module before it is installe
...
+
+While the REDCap Repo is the best way to manage the technical sharing of a module within the REDCap community, this doesn't quite meet the requirements of the academic community. Academics value publications and citations. Fortunately, there are tools to bridge the gap between a software repository and a citation. A digital object identifier or _DOI_ is a unique identifier for a document, dataset, image, software package, or other data type. Typically DOIs identify online objects. They are well-respected in the academic,  scientific, and governmental communities. There are online service providers that will generate, register, and curate DOIs for your creative works. One of these providers, [zenodo.org](https://zenodo.org/), has excellent integration with GitHub. Once you make the linkage between a GitHub repo and Zenodo, Zenodo will make a DOI and citation for the repo. Future GitHub releases will automatically get a new citable DOI in Zenodo. Cite these DOIs in academic articles, posters, grant applications, and sponsor reports, as you would with any other publication.
```
Comment (ChemiKyle) [inline docs/guide_for_admins_and_devs.md]:
```suggestion
While the REDCap Repo is the best way to manage the technical sharing of a module within the REDCap community, this doesn't quite meet the requirements of the academic community. Academics value publications and citations. Fortunately, there are tools to bridge the gap between a software repository and a citation. A digital object identifier or _DOI_ is a unique identifier for a document, dataset, image, software package, or other data type. Typically DOIs identify online objects. They are well-respected in the academic, scientific, and governmental communities. There are online service providers that will generate, register, and curate DOIs for your creative works. One of these providers, [zenodo.org](https://zenodo.org/), has excellent integration with GitHub. Once you make the linkage between a GitHub repo and Zenodo, Zenodo will make a DOI and citation for the repo. Future GitHub releases will automatically get a new citable DOI in Zenodo. Cite these DOIs in academic articles, posters, grant applications, and sponsor reports, as you would with any other publication.
```

Remove double space before scientific in "in the academic,  scientific, and governmental communities"

**Comment by pbchase** [comment]
All suggestions committed

**Pull request merged**